Web_Development/FrontEnd/HTML_Lists/Readme.md

```diff
@@ -0,0 +1,504 @@
+
+Now, we will see how to use these properties with examples.
+
+<br>
+
+## The List-Style Property
+
+<hr>
+
+The ***list-style-type*** property allows us to control the shape or style of marker in the list.
+
+For unordered lists possible values are :
+
+- none
+- disc
+- circle
+- square
+
+<br>
+
+**Example :**
+
+    ul
+      {
+    	list-style-type: none;
+      }
+
+This will style all unordered list to be displayed without any marker , instead of bullets.
+
+<br>
+
+For ordered lists possible values are :
+
+- decimal
+- decimal-leading-zero
+- lower-alpha
+- upper-alpha
+- lower-roman
+- upper-roman
+
+<br>
+
+**Example :**
+
+    ol
+      {
+    	list-style-type: decimal-leading-zero;
+      }
+
+This will style all ordered list to be displayed 0 prefixed numbers as marker , for single digit numbers.
+
+<br>
+<br>
+
+## The List-Style-Position Property
+
+<hr>
+By default, the marker of each list item is positioned outside of their display boxes.
+<br>
+<br>
+
+However, we can also position these markers or bullet points inside of the list item's display boxes using the ***list-style-position*** property along with the value inside.
+<br>
+<br>
+
+In this case the lines will wrap under the marker instead of being indented.
+
+<br>
+
+**Example :**
+
+    ol.in li
+    	  {
+    		 list-style-position: inside;
+    	  }
+    ol.out li
+    	  {
+    		 list-style-position: outside;
+       	  }
+
+<br>
+<br>
+
+## The List-Style-Image Property
+
+<hr>
+
+We can also set an image as a list marker using the property ***list-style-image***.
+
+**Example :**
+
+     ul li {
+    	     list-style-image: url("images/bullet.png");
+           }
+
+The style rule in the above code assigns a PNG image “bullet.png" as the list marker.
+
+<br>
+
+**<hr>**
+
+<br>
+
+# Setting All Lists Properties At Once
+
+The ***list-style*** property is a **shorthand** property for defining all the three properties : _list-style-type, list-style-image_, and *list-style-position* of a list in one place.
+
+<br>
+
+**Example :**
+   
+    ul
+     {
+        list-style: square inside url("images/bullet.png");
+     }
+
+<br>
+<br>
+
+# Creating Nav Bar Using List
+
+<br>
+
+<div align="center">
+<img  src="https://www.otallu.com/wp-content/uploads/2014/07/Create-a-simple-navigation-bar-with-CSS.jpg" />
+</div>
+<br>
+<br>
+
+HTML lists are frequently used to create horizontal navigation bar or menu that typically appears at the top of a website.
+
+<br>
+
+But since the list items are block elements, so to display them inline we need to use the ***CSS display property***.
+
+<br>
+
+**Example :**
+
+    ul
+       {
+    	  padding: 0;
+    	  list-style: none;
+    	  background: #f2f2f2;
+       }
+    ul li
+    	{
+    	   display:  inline-block;
+    	}
+    ul li a
+    	{
+    	display: block;
+    	padding: 10px 25px;
+    	color: #333;
+    	text-decoration: none;
+    	}
+    ul li a:hover
+    	{
+    	color: #fff;
+    	background: #939393;
+
```

Comment:
This block CSS ruleset should be closed. **(It's missing a closing bracket)**

Reply:
Okay, I will close it.